tests/coverage_analysis.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/sh
 
-set -e
```

Comment:
What is the reason for not stopping the script on errors?

Reply:
This was for debugging I'll change it